UnityProject/Assets/Scripts/Health/Sickness/Sickness.cs

```diff
@@ -145,7 +145,7 @@ public virtual void SymptompFeedback(LivingHealthMasterBase health)
 
 		public virtual bool CheckForCureInHealth(LivingHealthMasterBase health)
 		{
-			return health.CirculatorySystem.BloodPool.reagentKeys.Contains(CureForSickness) || health.CirculatorySystem.NutrimentToConsume.ContainsKey(CureForSickness);
+			return health.reagentPoolSystem.BloodPool.reagentKeys.Contains(CureForSickness);
```

Comment:
are nutriments in the same pool now?

Reply:
yeah, always been

Reply:
well, you removed 
``health.CirculatorySystem.NutrimentToConsume.ContainsKey(CureForSickness)`` this. That's why I ask

Reply:
idk why that was there 
NutrimentToConsume
that just the pre-Calculated amount of nutriment that the body consumes